networks/mainnet.json

```diff
@@ -190,30 +187,29 @@
             "description": "Standard Comptroller Impl"
         }
     },
-
     "Comp": {
         "contract": "Comp",
         "symbol": "COMP",
         "name": "Compound Governance Token",
         "decimals": 18,
         "address": "0xc00e94Cb662C3520282E6f5717214004A7f26888"
```

Comment:
Can we update the name from `"Compound Governance Token"` to `"Compound"`

Reply:
This is already on master this way. Are the underlying Tokens used in the app?

Reply:
Yeah, we used in the dapp to show the token name

Reply:
I agree though it is indeed wrong.